communications/newsletters/newsletter_12_Nov2019.md

```diff
@@ -0,0 +1,100 @@
+The participants of this meeting explored a number of topics in four themed sessions, draft report from which is available on [this HackMD](https://hackmd.io/zVTeKhG2SIiBLam1YtILNg).
+
+![]()
+
+## Relevant resources
+
+### Open Life Science Mentoring Program | Deadline Dec 8
+
+We are delighted to collaborate with the [Open Life Science Program](https://openlifesci.org) organised by Bérénice Batut, Yo Yehudi and Malvika Sharan as one of the [Mozilla Open Leader X projects](https://foundation.mozilla.org/en/opportunity/mozilla-open-leaders/).
+They are developing this mentoring program for mentoring and training early-stage researchers and potential academic leaders who want to become ambassadors for Life Science in their communities.
+The applications can be submitted via [EasyChair](https://easychair.org/cfp/ols-2020) until **December 8**.
+Reach out to the core team by emailing [hello@openlifesci.org](mailto:hello@openlifesci.org) or connect with them on Twitter [@openlifesci](https://twitter.com/openlifesci).
+
+## Tips & Tricks for new members
+
+### How to best cite Turing Way?
+
+Like our community members, Paola ([GitHub pull request](https://github.com/alan-turing-institute/the-turing-way/issues/648)) and Heidi ([Tweet](https://twitter.com/HeidiBaya/status/1189116056641658880)), have you also wondered how to cite Turing way?
+It is a fair question to ask when referring to an online Open Source project that undergoes several updates in the same week or sometimes on the same day. 
+
+All versions of this book (available under the [cc-by license]((http://creativecommons.org/licenses/by/4.0/legalcode))) can be cited using the DOI: [10.5281/zenodo.3233853](https://doi.org/10.5281/zenodo.3233853).
+This DOI represents all versions, and will always resolve to the latest one.
+Different versions can be found on [Zenodo](https://zenodo.org) and can be individually cited.
+
+(Kirstie's very sorry that she's behind with releasing some of the versions 😩!
+If you attended a book sprint or have written chapters for the book your name will appear in that author list soon.
+Promise!)
+
+## Acknowledgements and celebrations
+
+Kirstie Whitaker presented a talk at the Open Science Community Nijmegen on 9 October 2019 under the title [*The Turing Way*: A global collaboration for reproducible and collaborative research](https://zenodo.org/record/3478387#.XcmkDZNKjUI), and at NESTA Hack STIR event on 22 October 2019 under the title [*The Turing Way*: A handbook for reproducible research](https://zenodo.org/record/3515937#.XcmkeJNKjUI).
+
+Thanks to several of you for joining Kirstie Whitaker, Yo Yehudi, and Malvika Sharan at the [MozFest House](https://ti.to/Mozilla/mozfesthouse-whyspace/en) to discuss and work on *The Turing Way* specific ideas, and discussing the project with us at the [MozFest 2019](https://www.mozillafestival.org/en/).
+
+![]()
```

Comment:
What were you thinking of adding here? The pic from the steps?

Reply:
Your tweet with the new contributor who came and chatted with you.